public/content/en/basics/alias-strings.md

```diff
@@ -12,11 +12,11 @@ its content will never change again. And actually this is the second
 introduction: welcome UTF-8 `string`!
```

Comment:
The introduction "Now that we know what arrays are" uses "we", but I couldn't find a nice way to replace it.